deepl-connector-demo/src_hd/com/axonivy/connector/deepl/demo/translateTextAdvanced/translateTextAdvancedProcess.p.json

```diff
@@ -0,0 +1,119 @@
+{
+  "$schema" : "https://json-schema.axonivy.com/process/12.0.0/process.json",
+  "id" : "196146517BE8F821",
+  "kind" : "HTML_DIALOG",
+  "config" : {
+    "data" : "com.axonivy.connector.deepl.demo.translateTextAdvanced.translateTextAdvancedData"
+  },
+  "elements" : [ {
+      "id" : "f0",
+      "type" : "HtmlDialogStart",
+      "name" : "start()",
+      "config" : {
+        "signature" : "start",
+        "input" : {
+          "map" : {
+            "out.inputText" : "\"Hello world\""
```

Comment:
Small suggestion, the Demo Dialogs would be easier to use, if there are already values present, that work:
Therefore I'd already select values, that best underline your use-case:
- source language = EN
- target language = your favourit (ES? :blush: )
- tag-handling: html?

so that the dialog open like this: ... still users can customize, but for a demo it's easier to go with the defaults.
![Selection_008](https://github.com/user-attachments/assets/fa4bc164-4a19-4381-af31-0f543de5fa37)